src/GRA.Domain.Service/ActivityService.cs

```diff
@@ -1243,7 +1243,8 @@ public async Task<ServiceResult> UpdateFavoriteChallenges(IList<Challenge> chall
                     PointsEarned = 0,
                     IsDeleted = false,
                     AvatarBundleId = bundleId,
-                    Description = $"You unlocked the <strong>{bundle.Name}</strong> avatar bundle!"
+                    Description = $"You unlocked the <strong>{bundle.Name}</strong> avatar bundle! You can now see your new bundle on the Avatar's page!",
```

Comment:
Where is this displayed? Can it be made a link as above? (`<a asp-controller = \"Avatar\" asp-action=\"Index\">See your new bundle!</a>`)

Reply:
If my Avatar changes worked, we can probably simplify this to `Description = $"You unlocked the <strong>{bundle.Name}</strong> avatar bundle!"`